test/resources.test.js

```diff
@@ -219,21 +145,7 @@ describe('Resources query', function () {
 
         // Expect query to resemble: {"from":0,"size":50,"query":{"bool":{"filter":[{"bool":{"must_not":{"terms":{"nyplSource":["recap-hl"]}}}}]}},"sort":["uri"]}
         expect(body).to.be.a('object')
-        expect(body).to.have.deep.property('query', {
-          bool: {
-            filter: [
-              {
-                bool: {
-                  must_not: {
-                    terms: {
-                      nyplSource: ['recap-hl']
-                    }
-                  }
-                }
-              }
-            ]
-          }
-        })
+        expect(body).to.nested.include({ 'query.bool.filter[0].bool.must_not.terms.nyplSource[0]': 'recap-hl' })
```

Comment:
would this kind of assertion have worked with our previous code? or did the code change so now we can do this check?

Reply:
I think these "nested" assertions have been available for a while. They do help where you want to make a value assertion about a specific nested property and are okay with other aspects of the object changing